### PR TITLE
Fix toilet texture & fixture

### PR DESCRIPTION
**Pull request by MilenVolf**
## About the PR
The title

## Why / Balance
Passangers that got stuck between wall and a toilet will thank you.

### Before
Content Client_MmDrqfEl86
### After
Content Client_alOdrIlTBf
### OpenSeat
Content Client_ei4ZAcL0vA
Content Client_3ycbZvP5ro

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
N/A

### CODE REVIEW

**Comment by beck-thompson** [comment]
The amount of issues with toilets is kind of crazy! I need to look into it but their chems make no sense (I don't think). Its very unclear what they do in game as well because they have like 10 uses (Drain, disposals, hidden storage, a plunger somehow is involved??)

This good fix though!

File: Resources/Prototypes/Entities/Structures/Furniture/toilet.yml [hunk 1]
```diff
@@ -51,6 +51,15 @@
...
+  - type: Fixtures
+    fixtures:
+      fix1:
+        shape:
+          !type:PhysShapeCircle
+          radius: 0.4
+        density: 190
+        mask:
+        - TableMask
   - type: Construction
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Structures/Furniture/toilet.yml:63]: Why is it using tablemask and why does it not have a layer?

Comment (MilenVolf) [reply Resources/Prototypes/Entities/Structures/Furniture/toilet.yml:63]: > Why is it using tablemask and why does it not have a layer?

The fixture was copied from `SeatBase` prototype. It doesn't have mask neither.

Comment (MilenVolf) [reply Resources/Prototypes/Entities/Structures/Furniture/toilet.yml:63]: We can't go through the toilet with any of layers 'cause mob mask has `Impassable | HighImpassable | MidImpassable | LowImpassable` groups. So we don't need to have collision with any mob.
Mask obviously should be just MachineMask

**Comment by Emisse** [comment]
skibidi toiler

**Pull request merged**